docs/image-sizes.md

```diff
@@ -12,6 +12,16 @@ They have default dimensions, but you can override these either with code or in
 
 ![Media Settings](./assets/media-settings.png)
 
+### `content_width`
+
+If your theme defines a content width via the `$content_width` (or `$GLOBALS['content_width]`) global, this size will _override_ the size defined on the Settings > Media page. This will result in the Large size appearing to be the "wrong" size (using the size defined in the _global_ rather than the size in the admin) in the media modal.
+
+To make sure these sizes don't come into conflict, set the `$content_width` global to a large enough size that the admin setting won't override it.
+
+The `$content_width` variable has been a recommended part of a theme's `functions.php` file since WordPress 2.6.
```

Comment:
I think for safety's sake we should explicitly refer to it as the global eg. `$GLOBALS['content_width']` - my thinking is that it would avoid problems where someone is using or setting `$content_width` but forgetting to declare it as a variable. Alternatively I'd maybe add an `after_setup_theme` hook example showing declaring it as a global.

Reply:
None of the WordPress documentation references the `$GLOBALS` value at all, so I want to stick with what's already documented. That said, adding an example will drive this home, so adding that.